content/learn/workflow.md

```diff
@@ -0,0 +1,110 @@
+
+-   Once the submodule is ready, then images can be placed in the Markdown files using the figure shortcode. Other [shortcodes](/documentation/page-content/#use-shortcodes-to-add-features) are added where necessary.
+
+<div class="box tip">
+
+- See our [*Text-Editing Guide*](/learn/text-editing/) for the steps we follow to add content to a Quire project via text editor.
+
+</div>
+
+-   Any additional tweaks or customizations to the "first-pages" version of the publication can either be made in the `first-pages` branch or they can be made in task-specific branches. Task-specific branches should always be reviewed before being merged in the main `first-pages` branch. The review process gets a second set of eyes on the project to catch any potential errors. It’s helpful if the person reviewing has a good understanding of Quire and GitHub but it’s not required. Changes can be merged once they have been approved. Note: Changes are not merged into the `master` branch until the very end of the publication process.
+
+-   Once all the content has been added, formatting/customization is complete, and branches have been merged into the `first-pages` branch of the project is output for review. The **digital team** deploys a preview version of the website via Netlify and sends the URL, PDF, and hard copies of the publication to the **lead project editor**. Preview sites are always marked with an “uncorrected proof” banner.
+
+-   These files are then sent to the **proofreader**, **volume editors**, and **authors** for review.
+
+-   The **lead project editor** or **copy editor** collates all corrections onto the master hard copy set which marks the beginning of the "second-pages" stage.
+
+## Second-pages
+
+-   The **digital team** creates a `second-pages` branch in GitHub based off of the `first-pages` branch. Additional task or user-based branches should be made from, and therefore merged into, the `second-pages` branch.
+
+-   Any straightforward changes are entered in GitHub by either the **lead project editor** or **copy editor.** A printout of the manuscript with any complicated/outstanding edits marked with post-its and a memo outlining website issues are sent to the **digital team**.
+
+-   The **digital team** tackles these edits in Github. Anything that could not immediately be resolved is added to the repository’s Issue Tracker for more thorough review/problem-solving.
+
+-   Once all the "second-pages" changes are completed, the **digital team** deploys a another preview version of the website via Netlify and sends the URL, PDF, and hard copies of the publication to the **lead project editor**.
+
+-   Additionally, a print-on-demand version of the publication is printed and reviewed by **production** and the **digital team**. Special attention is paid to image quality.
+
+## Final-pages
+
+-   A final round of review takes place marking the "final-pages phase. All changes are then merged into the `master` branch on Github.
```

Comment:
final pages needs tick marks (`) to get the formatting. Has an errant double-quote now (").